src/components/dialogs/network-modifications/widget/AccordionIllustration.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { FunctionComponent, PropsWithChildren, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import {
+    Accordion,
+    AccordionDetails,
+    AccordionSummary,
+    Theme,
+    Typography,
+} from '@mui/material';
+import { ExpandCircleDown, ExpandMore } from '@mui/icons-material';
+
+type AccordionIllustrationProps = {
+    state: boolean;
+    onClick: (show: boolean) => void;
+};
+
+export const styles = {
+    accordion: {
+        background: 'none',
+    },
+    accordionIllustrationSummary: (theme: Theme) => ({
+        alignContent: 'flex-start',
+        flexDirection: 'row',
+        flexGrow: 0,
+        justifyContent: 'left',
+        transition: '0.2s',
+        '& .MuiAccordionSummary-expandIconWrapper': {
+            transform: 'rotate(180deg)',
+        },
+        '& .MuiAccordionSummary-expandIconWrapper.Mui-expanded': {
+            transform: 'rotate(0deg)',
+        },
+        '& .MuiAccordionSummary-content': {
+            marginLeft: theme.spacing(0),
+            flexGrow: 0,
+            msFlexPositive: 0,
```

Comment:
- msFlexPositive will be added automatically by emotion if you add flexGrow.
- do we really need justify-content ?

Reply:
Cleaned.